i18n/es.jsonp.js

```diff
@@ -137,6 +139,7 @@ document.localeJson = {
       "dialog-article-readerror-title": "Error al leer artículo",
       "dialog-cancel": "Cancelar",
       "dialog-ok": "OK",
+      "dialog-hide": "No vuelvas a preguntar",
```

Comment:
We're using the forma (Usted) form of you in this app, so this needs to be "No vuelva a preguntar".

Reply:
I used an online translation tool and wasn't really sure about it. 😅
Thanks for the clarification! Updated in [1c1869b](https://github.com/kiwix/kiwix-js/pull/1236/commits/1c1869b9ea6b9074457731f7c2d9d9bb14c30078).